lib/context.js

```diff
@@ -58,6 +61,28 @@ class Context {
   get isBot() {
     return this.payload.sender.type === 'Bot';
   }
+
+  /**
+   * Reads the plugin configuration from the given YAML file in the `.github`
+   * directory of the repository.
+   *
+   * @example
+   *
+    // Load config from .github/myplugin.yml in the repository
+    const config = await context.config('myplugin.yml');
+
+    if(config.close) {
+      context.github.issues.edit(context.issue({state: 'closed'}));
+    }
+   *
+   * @param {string} fileName Name of the YAML file in the `.github` directory
+   * @return {Promise<Object>} Configuration object read from the file
+   */
+  async config(fileName) {
+    const params = this.repo({path: path.join('.github', fileName)});
```

Comment:
Do you think it's necessary to make `.github` the default location for files code wise? I understand why we recommend that it be used, but as you say in the docs, what if someone had some other similar convention? Is it really that much worse to make people establish the path as `.github/my_file.yml`? Or do you think it's worth it to force people to swap to this convention?